src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java

```diff
@@ -137,22 +137,19 @@ private static Credential authorize() throws Exception {
             OAuth2Browser.INSTANCE).authorize("user");
     }
 
-    private static InputStream getSecret() throws IOException {
-        String file = System.getProperty("google.client.secret");
-        if (file != null) {
-            return new FileInputStream(file);
-        }
-
-        String portableDir = System.getProperty("weasis.portable.dir");
-        if (portableDir != null) {
-            File portableSecrets = new File(portableDir, SECRETS_FILE_NAME);
-            if (portableSecrets.exists() && !portableSecrets.isDirectory()) {
-                return new FileInputStream(portableSecrets);
-            }
-        }
-
-        return GoogleAPIClient.class.getResource("/" + SECRETS_FILE_NAME).openStream();
+  private static InputStream getSecret() throws IOException {
+    String extendedProperties = System.getProperty("felix.extended.config.properties")
```

Comment:
You have changed the name of the field here, we should probably also change this in the documentation..

Reply:
I changed this part because of `BundleTools.SYSTEM_PREFERENCES.getProperty` doesn't work. Also, we can remove the .weasis folder to remove the previously cached plugin. I think that adding client_secret.json to the properties folder can be easy for users.

Reply:
Ok sounds good thanks Artyom.